Get-help.md

```diff
@@ -0,0 +1,141 @@
+
+### Email
+
+We have several mailing lists in the form of Google Groups that you can join:
+
+* [oppia-announce](https://groups.google.com/forum/#!forum/oppia-announce) is for announcements of new releases or blog posts. It's not for asking questions though.
+* [oppia-dev](https://groups.google.com/forum/#!forum/oppia-dev) is the main mailing list for communication between developers and for technical questions. You can post to it even if you're not a member of the group. This is where you can ask questions, solicit feedback, or make developer-specific announcements (e.g. a temporary GitHub outage).
+
+You can also email your mentor with any questions. If you don't have a mentor, complete the steps on the [[wiki page for contributing code to Oppia|Contributing-code-to-Oppia]].
+
+### Gitter
+
+We have a developer chat room [here](https://gitter.im/oppia/oppia-chat). Feel free to drop in and say hi! Oppia's Welfare team monitors this chat to help out new contributors, so it's the best place to ask questions about setting up Oppia or getting started.
+
+You can also directly message developers over Gitter. However, some developers don't monitor their Gitter messages regularly, so you might not get a response very quickly (or at all).
+
+### Google Chat / Hangouts
+
+Most Oppia developers and teams use Google Chat or Google Hangouts as their primary means of communication, and they usually respond quickly.  However, invites often get lost in spam folders, and some developers use a non-public email address for Google Chat. You can ask your mentor to put you in touch with a developer if you don't know their address or if they haven't acted on your invite.
+
+### GitHub
+
+If you have a question about a pull request or issue, you can also reach out to developers by at-mentioning them (e.g. `@developer-username`) in a comment and assigning them to the issue. Be sure you both at-mention and assign them! Some developers only look at their GitHub notifications (which at-mentions trigger), while others only look at what they're assigned to.
+
+You can even mention whole teams of people! For example, if you find an issue that is destabilizing the project, you could notify all the core maintainers by including `@oppia/core-maintainers` in your issue. The teams are all listed [here](https://github.com/orgs/oppia/teams).
+
+## How to Ask Good Questions
+
+At Oppia we don’t care how silly your question is! Just ensure your question is clear, and provide us enough information to help us resolve it faster. We've divided the questions into 2 categories - Setup-related and General questions. You can start following the sections below to understand how you can ask each of them.
+
+### Setup-related questions
+
+**Helpful advice**: You can choose any OS you’re comfortable with. However, most contributors use Linux or MacOS. Getting Oppia up and running on Windows can be a challenge.
```

Comment:
With the new Linux subsystem, is this still the case? /cc @aks681 @EricZLou

Reply:
I haven't had any issues but I haven't been checking Gitter to see if others have

Reply:
We already have a note in the installation guides that installing on Windows is more complicated, so I'll remove this line.

Reply:
done